src/sql/src/statements/copy.rs

```diff
@@ -152,7 +152,7 @@ mod tests {
             Statement::Copy(copy) => {
                 let new_sql = format!("{}", copy);
                 assert_eq!(
-                    r#"COPY tbl FROM s3://my-bucket/data.parquet WITH (format = 'parquet', pattern = '.*parquet.*') CONNECTION (region = 'us-west-2', secret_access_key = '******')"#,
+                    r#"COPY tbl FROM s3://my-bucket/data.parquet WITH (format = 'parquet', pattern = '.*parquet.*') CONNECTION (region = 'us-west-2', secret_access_key = Secret([REDACTED alloc::string::String]))"#,
```

Comment:
Actually I don't find this is better than `'******'`, perhaps we can still display `SecretString` as `'******'`?